discovery-provider/prod.env

```diff
@@ -49,4 +49,4 @@ audius_delegate_private_key=
 audius_discprov_url=
 
 # chain 
-audius_discprov_max_signers=1
\ No newline at end of file
+audius_discprov_max_signers=0
```

Comment:
n00b question, what exactly does this control?

Reply:
nah all good this is an env var to say how many other DN signers we want besides foundation. when the chain starts it's just foundation but they'll propose others up to the max signers. in this case i just want to restart prod without proposing others.

Reply:
https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/src/tasks/index_network_peers.py#L56